src/components/Auth/Details.tsx

```diff
@@ -45,6 +89,60 @@ const Details = () => {
             />
           </LI>
         </ul>
+        <H3>Bookmarked Content</H3>
```

Comment:
Same comments about data feching, retrying and error states as on weeklist component

Reply:
I'm thinking this should be done differently for the Details page. On the week list component, we want all of the weeks to load, regardless of their status. For the Details page, I would assume that we only want the bookmarked content. So instead of showing them a full list of content, that they haven't bookmarked (while data is loading) let's just show the loader and a message saying "Loading additional data...". After some attempts, the user can click the button to try again to load their bookmarked data.